build.sbt

```diff
@@ -4,6 +4,43 @@ import Common._
 import com.typesafe.tools.mima.core._
 import com.typesafe.tools.mima.plugin.MimaKeys.{ mimaBinaryIssueFilters, mimaPreviousArtifacts }
 
+// Customise sbt-dynver's behaviour to make it work with tags which aren't v-prefixed
```

Comment:
Would rather keep it in `version.sbt`

Reply:
If you insist, ok, we can keep it there. (In all the other Play repos we removed `version.sbt` to make clear we now use sbt-dynver, and also keep the dynver configs in `build.sbt`)
Long term solution however IMHO is that we add a config to sbt-dynver to achieve what you did here: https://github.com/sbt/sbt-dynver/issues/227 (if you want you can provide a PR to sbt-dynver 😉  )

Reply:
I don't see why having everything in `build.sbt` rather than modular files would make it clear?

Reply:
@anorm Reverted here: #461